cmd/tink-worker/action.go

```diff
@@ -39,10 +39,10 @@ func executeAction(ctx context.Context, action *pb.WorkflowAction, wfID string)
 	var cancel context.CancelFunc
```

Comment:
```
       timeCtx = ctx
	if action.Timeout > 0 {
                var cancel context.CancelFunc
		timeCtx, cancel = context.WithTimeout(context.Background(), time.Duration(action.Timeout)*time.Second)
		defer cancel()
	}
```
Something like this sounds a bit easier to read according to me.

Reply:
Done